code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -44,6 +44,7 @@
 	var/bleedsuppress = 0 //for stopping bloodloss, eventually this will be limb-based like bleeding
 
 	var/name_override //For temporary visible name changes
+	var/genital_override = FALSE //Force genitals on things incase of chems

```

Comment:
As long as this has the opt-in overarching check for no arousal = no genitals at all, this should still be fine.